css/styles.css

```diff
@@ -64,6 +72,18 @@ textarea {
   border-radius: 4px;
 }
 
+.entry {
+  margin-bottom: 2rem;
+}
+
+ul {
+  list-style: none;
+}
+
+.new {
+  margin-left: 37rem;
+}
```

Comment:
Woah, this shouldn't need to be done with margin. I would make the container a flexbox and use `justify-content` to put space between the children